dm: enhance description about error triggering condition (#11620)
pingcap/docs: dm/dm-handle-alerts.md

- Description:

>     When the relay log processing unit encounters a non-autorecoverable error (for example, binlog file not found), or when it encounters multiple recoverable errors in a short period of time (for example, network problems), this unit moves to `Paused` state, and an alert is triggered immediately.

Author: ```suggestion
    When the relay log processing unit encounters a non-autorecoverable error (for example, binlog file not found), or when it encounters multiple recoverable errors in a short period of time (for example, network problems), this alert is triggered immediately.
```

>     When the relay log processing unit encounters a non-autorecoverable error (for example, binlog file not found), or when it encounters multiple recoverable errors (for example, network problems) in a short period of time (for example, more than 3 times in 2 minutes), this alert is triggered immediately.

Reviewer: ```suggestion
    When the relay log processing unit encounters a non-autorecoverable error (for example, binlog files not found), or when it encounters multiple recoverable errors (for example, network problems) in a short period of time (for example, more than 3 times in 2 minutes), this alert is triggered.
```

Reviewer: Please also update other places for this sentence.

- Description:

>     When the Dump processing unit encounters a non-autorecoverable error (for example, binlog file not found), or when it encounters multiple recoverable errors in a short period of time (for example, network problems), this unit moves to the `Paused` state, and an alert is triggered immediately.

Author: ```suggestion
    When the Dump processing unit encounters a non-autorecoverable error (for example, binlog file not found), or when it encounters multiple recoverable errors in a short period of time (for example, network problems), this alert is triggered immediately.
```

- Description:

>     When the Load processing unit encounters a non-autorecoverable error (for example, binlog file not found), or when it encounters multiple recoverable errors in a short period of time (for example, network problems), this unit moves to the `Paused` state, and an alert is triggered immediately.

Author: ```suggestion
    When the Load processing unit encounters a non-autorecoverable error (for example, binlog file not found), or when it encounters multiple recoverable errors in a short period of time (for example, network problems), this alert is triggered immediately.
```

- Description:

>     When the binlog replication processing unit encounters a non-autorecoverable error (for example, binlog file not found), or when it encounters multiple recoverable errors in a short period of time (for example, network problems), this unit moves to the `Paused` state, and an alert is triggered immediately.

Author: ```suggestion
    When the binlog replication processing unit encounters a non-autorecoverable error (for example, binlog file not found), or when it encounters multiple recoverable errors in a short period of time (for example, network problems), this alert is triggered immediately.
```
